app/mappers/grades_by_year.rb

```diff
@@ -0,0 +1,14 @@
+class GradesByYear < ROM::Mapper
+  register_as :grades_by_year
+  relation :grade_years
+
+  model GradeYear
```

Comment:
lol yeah a necessary evil... I figure that as a code base transitions from AR to ROM this will have to happen. However then when `GradeYear` is no longer using any AR methods, then it can just be converted to a poro. A slow but, hopefully, incremental process.